semcore/textarea/src/style/textarea.shadow.css

```diff
@@ -55,4 +56,17 @@ STextarea[state='invalid'] {
   &:focus {
     box-shadow: var(--intergalactic-keyboard-focus-invalid, 0px 0px 0px 3px rgba(255, 73, 83, 0.5));
   }
+}
+
+SContainer {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  margin-bottom: 4px;
```

Comment:
Probably instead of `4px` you should use spacing design token

Reply:
no, this is space after textarea, which I removed by `vertical-align: top;`

Reply:
removed